lib/fluent/plugin/in_windows_eventlog2.rb

```diff
@@ -40,6 +40,9 @@ class ReconnectError < Fluent::UnrecoverableError; end
     config_param :read_from_head, :bool, default: false, deprecated: "Use `read_existing_events' instead."
     config_param :read_existing_events, :bool, default: false
     config_param :parse_description, :bool, default: false
+    config_param :description_prefix_word_delimiter, :string, default: "."
```

Comment:
How about the option name `description_key_delimiter`?
(I couldn't understand what `prefix word` means. Please let me know if I misunderstand something.)

```suggestion
    config_param :description_key_delimiter, :string, default: "."
```

Reply:
Good idea! I agree that this would make more sense in application.